tests/phpunit/agentic-commerce/class-wc-stripe-agentic-shipping-calculator-test.php

```
@@ -118,6 +124,184 @@ public function test_matching_zone_returns_valid_shipping_options( float $cost,
+			}
+			$raw_items[] = $raw_item;
+		}
+
+		$result = $this->calculator->calculate( $this->build_event_from_raw_items( $raw_items ), 'usd' );
+
+		$this->assertSame( $expected_amount, $result['shipping_options'][0]['shipping_rate_data']['fixed_amount']['amount'] );
+
+		foreach ( $products as $product ) {
+			$product->delete( true );
+		}
+	}
```

PHPMD (2.15.0)

[error] The method test_cost_expression_prices_package_contents is not named in camelCase. (undefined)

(CamelCaseMethodName)